docs/data/material/components/button-group/LoadingButtonGroup.tsx

```diff
@@ -1,17 +1,16 @@
 import * as React from 'react';
 import ButtonGroup from '@mui/material/ButtonGroup';
 import Button from '@mui/material/Button';
-import LoadingButton from '@mui/lab/LoadingButton';
 import SaveIcon from '@mui/icons-material/Save';
 
 export default function LoadingButtonGroup() {
   return (
     <ButtonGroup variant="outlined" aria-label="Loading button group">
       <Button>Submit</Button>
-      <LoadingButton>Fetch data</LoadingButton>
-      <LoadingButton loading loadingPosition="start" startIcon={<SaveIcon />}>
+      <Button>Fetch data</Button>
```

Comment:
This button should include the `loading` prop.

```suggestion
      <Button loading>Fetch data</Button>
```